efsity/src/main/java/org/smartregister/command/PublishFhirResourcesCommand.java

```diff
@@ -86,12 +95,20 @@ void setProperties(Properties properties){
         }
     }
 
-    void publishResources() throws IOException {
+    void publishResources() throws IOException, ValidationException, GenerationException {
         ArrayList<String> resourceFiles = getResourceFiles(projectFolder);
         ArrayList<JSONObject> resourceObjects = new ArrayList<>();
+        boolean validateResourceBoolean = Boolean.parseBoolean(validateResource);
+
         for(String f: resourceFiles){
+            if (validateResourceBoolean) {
+                FctUtils.printInfo(String.format("Validating file \u001b[35m%s\u001b[0m", f));
+                ValidateFhirResourcesCommand.validateFhirResources(f);
+            } else {
+                FctUtils.printInfo("Publishing Without Validation");
```

Comment:
Maybe add the fileName to this is well? Might be useful when debugging to easily see which files were published without validation instead of just that statement over and over if multiple files